hal/src/argon/bootloader_platform_12.c

```diff
@@ -1,2071 +1,4 @@
 const unsigned char bootloader_platform_12_bin_miniz[] = {
-  0xe5, 0xb2, 0x7d, 0x7c, 0x53, 0xe5, 0xfd, 0x3f, 0x7c, 0x5d, 0xe7, 0x9c,
```

Comment:
Let's leave a TODO somewhere (maybe in the release runbook) that we'll need to bump the bootloader dependency version.